tests/cupyx_tests/scipy_tests/signal_tests/test_wavelets.py

```diff
@@ -10,7 +9,8 @@
     pass
 
 
-@testing.with_requires('scipy')
+@pytest.mark.filterwarnings('ignore::DeprecationWarning')
+@testing.with_requires("scipy<1.15")
 class TestWavelets:
     @testing.with_requires('scipy<1.12.0')
```

Comment:
offline discussion: Remove `scipy<1.12.0` restrictions to run tests on `scipy<1.15`